packages/compass-sidebar/src/components/sidebar-instance-stats/sidebar-instance-stats.jsx

```diff
@@ -55,6 +55,7 @@ class SidebarInstanceStats extends PureComponent {
     }
 
     return (
+      /* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
```

Comment:
Tiny nit: if we have a ticket to address those later I'd really prefer we add some TODOs right in the code or create a ticket, these are super low hanging fruits to improve compass usability and I'd prefer that these particular eslint-disables would stay in code as short as possible

Reply:
https://jira.mongodb.org/browse/COMPASS-5918